package.json

```diff
@@ -90,7 +90,7 @@
     "flexboxgrid": "^6.3.0",
     "immutable": "^3.8.1",
     "lodash": "^4.5.1",
-    "lzma-native": "^1.5.2",
+    "lzma-native": "^2.0.2",
```

Comment:
IIRC this updated version of `lzma-native` also allows us to use a newer version of `node-gyp` (or whatever the relevant package is called - hopefully you know what I mean), rather than locking ourselves to an older version?

Perhaps this change is worth doing anyway in a separate PR against the `master` branch?